### DISCUSSION TITLE
[embedded-sdk] Open a different dashboard on click from an already embedded dashboard

### DISCUSSION THREAD

**Discussion by DheerajMahra** [apache/superset#27050] [Q&A / Help]
Hi,

I have embedded a dashboard using `embedDashboard` api provided by `@superset-ui/embedded-sdk` inside my react app.

React App and dashboard iframe have different domains.

React App: `<react-domain>`
Dashboard: `<dashboard-domain>`

<img width="641" alt="Screenshot 2024-02-08 at 4 24 06 PM" src="https://github.com/apache/superset/assets/39774163/f331798c-df79-472f-961c-d64d51dcd92e">

The dashboard has some tiles that redirect to different dashboards. Tiles have anchor tag with href `<dashboard-domain>/superset/dashboard/<dashboard-embedded-id>`

On click of the tile, I get redirected to the superset login page and not to the dashboard inside the iframe.

**Comment by simondo92** [comment]
Hey @DheerajMahra, 

Did you find a solution for this, or get any answers?
We're struggling with the same problem. 

Thanks
